Xamarin.Forms.Platform.Android/Extensions/TextViewExtensions.cs

```diff
@@ -7,41 +8,65 @@ namespace Xamarin.Forms.Platform.Android
 {
 	internal static class TextViewExtensions
 	{
-		public static void SetLineBreakMode(this TextView textView, LineBreakMode lineBreakMode)
+		public static void SetMaxLines(this TextView textView, Label label)
 		{
+			var maxLines = label.MaxLines;
+
+			if (maxLines == (int)Label.MaxLinesProperty.DefaultValue)
+			{
+				// MaxLines has not been explicitly set, so just let it be whatever was set by LineBreakMode
+				return;
```

Comment:
What happens if I set MaxLines to, say, 5 and then set it back to -1? Should we call SetLineBreakMode() here?

Reply:
Ah, good point. I'll fix that.